src/Components/TaskManagerComponent/TaskManagerComponent/WebApi.ts

```diff
@@ -16,7 +22,12 @@ export class WebApi implements IWebApi {
     }
 
     retrieveOptionSetMetadata(entityType: string, attributeName: string): Promise<Response> {
-        return window.fetch(`${this.clientUrl}/api/data/v9.1/EntityDefinitions(LogicalName='${entityType}')/Attributes(LogicalName='${attributeName}')/Microsoft.Dynamics.CRM.${attributeName === "statuscode" ? "StatusAttributeMetadata" : "PicklistAttributeMetadata"}?$select=LogicalName&$expand=OptionSet($select=Options)`,
+        return window.fetch(
+            `${
+                this.clientUrl
+            }/api/data/v9.1/EntityDefinitions(LogicalName='${entityType}')/Attributes(LogicalName='${attributeName}')/Microsoft.Dynamics.CRM.${
+                attributeName === "statuscode" ? "StatusAttributeMetadata" : "PicklistAttributeMetadata"
+            }?$select=LogicalName&$expand=OptionSet($select=Options)`,
```

Comment:
nitpick: We might want to format this differently / ignore the formatting on the line to make it more readable.